cmd/soci/commands/push.go

```diff
@@ -102,7 +102,6 @@ if they are available in the snapshotter's local content store.
 			if err != nil {
 				return fmt.Errorf("cannot create OCI local store: %w", err)
 			}
-
 			indexDesc := indexDescriptors[len(indexDescriptors)-1]
```

Comment:
I feel like I must be missing something here, but your code looks like it sorts oldest to newest (which should work here), but the example in the description looks like it's printing newest to oldest. How do they both work?

Reply:
The `list` command does not use `GetIndexDescriptorCollection` like `push` does; instead, it directly talks to the artifacts db. The sorting is done in `GetIndexDescriptorCollection`.